freqtrade/constants.py

```diff
@@ -260,7 +260,7 @@
                                      'enum': TELEGRAM_SETTING_OPTIONS,
                                      'default': 'off'
                                      },
-                        'sell': {'type': 'string', 'enum': TELEGRAM_SETTING_OPTIONS},
+                        'sell': {'type': 'object'},
```

Comment:
Unless there's a good reason not to - i'd want to check the settings within "sell" as well.

By accepting `object` - we allow "random" values (both as keys, and as values) - and history has shown that users have a tendency to misstype things - so you'd get `slient` instead of `silent` - and wonder why it doesn't work.

While we can't really prevent misstyped keys - we can prevent it on values.